galaxy_skeleton2.R

```diff
@@ -16,7 +16,8 @@ library(tidyverse)    # Contains most of what we need.
 # the file does not end with an "end of line"-character (EOL). This does not
 # seem to pose a problem later, and it seems that we can silece the warning by
 # switchin off the "warn"-argument. Do that if you wish.
-raw_file <- readLines(con = "?")
+raw_file <- readLines(con = "/Users/siljebui/Public/Master/BAN400/git-sbsilje/
```

Comment:
Use relative paths. I cannot load this file without manually changing the path. Since the file is in the same directory as your script, you could simply write "suites_dw_Table1.txt".